### ISSUE TITLE
My Home: Update controller to new useLaunchpad and fetchLaunchpad methods

### ISSUE THREAD

**Issue by edanzer** [Automattic/wp-calypso#76472]
### Details

This PR has two related parts. 
   - The my home controller is using fetchLaunchpad from calypso/data/sites. This should be replaced with fetchLaunchpad from data-stores. This change can now be made at any time. 
   - The my Home controller is using fetchLaunchpadChecklist from the help center package. This should be replaced with fetchLaunchpad from data-stores. **Before we can make this change, our update to return checklists from the main Launchpad endpoint needs to be deployed to production for jetpack-mu-com.**

**Comment by jeyip** [comment]
> The my home controller is using fetchLaunchpad from calypso/data/sites. This should be replaced with fetchLaunchpad from data-stores. This change can now be made at any time.

Addressed in https://github.com/Automattic/wp-calypso/pull/76514

> The my Home controller is using fetchLaunchpadChecklist from the help center package. This should be replaced with fetchLaunchpad from data-stores. Before we can make this change, our update to return checklists from the main Launchpad endpoint needs to be deployed to production for jetpack-mu-com.

Will be addressed in a separate PR

**Comment by edanzer** [comment]
Done with https://github.com/Automattic/wp-calypso/pull/76514
